Dockerfile

```diff
@@ -101,10 +101,13 @@ VOLUME [ \
   "/opt/scripts" \
 ]
 
+# Use README for the help command
+ADD README.md /usr/share/man/help.txt
```

Comment:
It's more consistent with the previous `help` function and the Markdown is not parsed/rendered so basically it's displayed as a text file, hence the extension